stepactions/bundles/get-unreleased-bundle/0.1/get-unreleased-bundle.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: tekton.dev/v1beta1
+kind: StepAction
+metadata:
+  name: get-unreleased-bundle
+spec:
+  description: >-
+    This StepAction retrieves the highest bundle version from a specified package and channel in the provided FBC fragment.
+    If package name and/or channel name are not specified, the step will determine the default package and/or channel name.
+    Default package name determination:
+      * If there is only one 'olm.package', it's name is returned
+      * If multiple 'olm.package' entries contain unreleased bundles, user input is required; the PACKAGE_NAME parameter must be set by the user
+    Default Channel Name Determination:
+      * The default channel name corresponds to the 'defaultChannel' entry of the selected package
+    The StepAction checks whether the bundle is unreleased, specifically, not present in the Red Hat production Index Image (registry.redhat.io/redhat/redhat-operator-index).
```

Comment:
As above I would keep "highest bundle version" wording: `The StepAction checks whether the highest bundle version is unreleased`